src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+/**
+ * Class DukeException represents an exception resulted due to wrong format of command
+ * entered into Danh"s Duke
+ */
+class DukeException extends Exception {
+    public DukeException(String errorMessage) {
+        super(errorMessage);
+    }
+}
```

Comment:
I believe checkstyle prefers if there is an additional empty line after the last parentheses in your files. You may want to check up on that.